lib/template.html

```diff
@@ -206,28 +209,57 @@
     "headline": {{ seo_page_title | jsonify }},
 {% endif %}
 
+{% if seo_author_name %}
+    "author": {{ seo_author_name | jsonify }},
+{% endif %}
+
 {% if seo_page_image %}
-    "image": {{ seo_page_image | jsonify }},
+    {% if page.image.height && page.image.width %}
+      "image": {
+          "@type": "ImageObject",
+          "url": {{ seo_page_image | jsonify }},
+          "height": {{ page.image.height | jsonify }},
+          "width": {{ page.image.width | jsonify }}
+      },
+    {% else %}
+        "image": {{ seo_page_image | jsonify }},
+    {% endif %}
 {% endif %}
 
 {% if page.date %}
     "datePublished": {{ page.date | date_to_xmlschema | jsonify }},
 {% endif %}
 
+{% if page.dateModified %}
+    "dateModified": {{ page.dateModified | date_to_xmlschema | jsonify }},
+{% elseif page.date %}
+    "dateModified": {{ page.date | date_to_xmlschema | jsonify }},
+{% endif %}
+
```

Comment:
Add a date modified field. Useful when wanting to control this manually. Like when I make a change to the markup file, but I want to override the timestamp given.

Will default to page.date if never "modified". Will not exist if neither field present.